### PR TITLE
Lps 87433 88135

### PR DESCRIPTION
**Pull request by balazssk**
Hi @achaparro ,
I've made the changes to the verify->upgrade processes we've discussed. (https://issues.liferay.com/browse/LPS-87433)
I've also found the reason behind the Calendar portlet adding the instance id in the typesettings and made a fix for that as well. (https://issues.liferay.com/browse/LPS-88135)
Can you please review it?

Thanks,

### CODE REVIEW

**Comment by achaparro** [comment]
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Comment by achaparro** [comment]
Hi @balazssk,

I have reviewed the pull and added some changes. During the tests I have found that we get the following for the principal site:
```
staged-portlet_com_liferay_blogs_web_portlet_BlogsAdminPortlet=false
staged-portlet_com_liferay_bookmarks_web_portlet_BookmarksAdminPortlet=false
staged-portlet_com_liferay_bookmarks_web_portlet_BookmarksPortlet=true
staged-portlet_com_liferay_calendar_web_portlet_CalendarPortlet=false
staged-portlet_com_liferay_document_library_web_portlet_DLAdminPortlet=true
staged-portlet_com_liferay_dynamic_data_lists_web_portlet_DDLPortlet=false
staged-portlet_com_liferay_dynamic_data_mapping_web_portlet_PortletDisplayTemplatePortlet=false
staged-portlet_com_liferay_journal_web_portlet_JournalPortlet=true
staged-portlet_com_liferay_knowledge_base_web_portlet_AdminPortlet=false
staged-portlet_com_liferay_message_boards_web_portlet_MBAdminPortlet=false
staged-portlet_com_liferay_mobile_device_rules_web_portlet_MDRPortlet=true
staged-portlet_com_liferay_polls_web_portlet_PollsPortlet=false
staged-portlet_com_liferay_wiki_web_portlet_WikiAdminPortlet=false
staged-portlet_com_liferay_wiki_web_portlet_WikiPortlet=false
```

And the following for the global group (it was empty initially):
```
staged-portlet_com_liferay_blogs_web_portlet_BlogsAdminPortlet=false
staged-portlet_com_liferay_bookmarks_web_portlet_BookmarksAdminPortlet=false
staged-portlet_com_liferay_document_library_web_portlet_DLAdminPortlet=false
staged-portlet_com_liferay_message_boards_web_portlet_MBAdminPortlet=false
staged-portlet_com_liferay_wiki_web_portlet_WikiAdminPortlet=false
```

I am currently checking why this is happening, do you have any clue?

Thanks.

**Comment by achaparro** [comment]
Pull request submitted to https://github.com/brianchandotcom/liferay-portal/pull/65807. <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Comment by achaparro** [comment]
Pull request submitted to https://github.com/brianchandotcom/liferay-portal/pull/65808. <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

File: modules/apps/calendar/calendar-web/src/main/java/com/liferay/calendar/web/internal/upgrade/v_1_2_0/UpgradeStagingGroupTypeSettings.java [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+		Pattern pattern = Pattern.compile(
+			_STAGED_CALENDAR_PORTLET + _INSTANCE_SEPARATOR + "\\w{12}",
+			Pattern.MULTILINE);
+
+		Matcher matcher = pattern.matcher(typeSettings);
+
+		return matcher.replaceAll(_STAGED_CALENDAR_PORTLET);
+	}
+
+	private void _removeStagedCalendarInstanceId() throws Exception {
```
Comment (achaparro) [inline modules/apps/calendar/calendar-web/src/main/java/com/liferay/calendar/web/internal/upgrade/v_1_2_0/UpgradeStagingGroupTypeSettings.java]: Hi @balazssk,

What we need to analyze is where the upgrade process adds the sufix _INSTANCE_XXX to the calendar name in the typeSettings for the group record, w ehave to fix that. During my tests before starting the upgrade there wasn't an INSTANCE_ sufix in that field.

File: modules/apps/calendar/calendar-web/src/main/java/com/liferay/calendar/web/internal/upgrade/v_1_2_0/UpgradeStagingGroupTypeSettings.java [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+		StringBundler sb = new StringBundler(5);
+
+		sb.append("select groupId, typeSettings from Group_ where ");
```
Comment (achaparro) [inline modules/apps/calendar/calendar-web/src/main/java/com/liferay/calendar/web/internal/upgrade/v_1_2_0/UpgradeStagingGroupTypeSettings.java]: @balazssk  we can't execute direct SQL for tables which don't belong to the module. That's the golden rule::
https://grow.liferay.com/share/DXP-+How+to+code+upgrade+process+and+backport+rules

File: modules/apps/calendar/calendar-web/src/main/java/com/liferay/calendar/web/internal/upgrade/v_1_2_0/UpgradeStagingGroupTypeSettings.java (hunk 1 above)
Comment (balazssk) [reply modules/apps/calendar/calendar-web/src/main/java/com/liferay/calendar/web/internal/upgrade/v_1_2_0/UpgradeStagingGroupTypeSettings.java]: yes, if I remember correctly it gets added as the Calendar portlet uses the instanceable upgrade from BaseUpgradePortletId while the other portlets are using the uninstanceable.

File: modules/apps/calendar/calendar-web/src/main/java/com/liferay/calendar/web/internal/upgrade/v_1_2_0/UpgradeStagingGroupTypeSettings.java (hunk 2 above)
Comment (balazssk) [reply modules/apps/calendar/calendar-web/src/main/java/com/liferay/calendar/web/internal/upgrade/v_1_2_0/UpgradeStagingGroupTypeSettings.java]: oops, guess creating something similar to the non-admin->admin upgrade process can be done here as well.

**Pull request closed without merging**